app/Http/Controllers/App/EmailController.php

```
@@ -191,6 +192,22 @@ public function move(Dropbox $dropbox, DropboxMail $mail, Dropbox $newDropbox):
         return redirect(route('app.email.index', ['dropbox' => $dropbox->id]));
     }
 
+    public function snooze(DropboxMailSnoozeRequest $request, Dropbox $dropbox, DropboxMail $mail): RedirectResponse
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$dropbox'. (undefined)

(UnusedFormalParameter)